src/main/java/com/ericsson/ei/frontend/utils/BackEndInstanceFileUtils.java

```diff
@@ -46,10 +60,37 @@ public void init() throws IOException {
             }
 
             setEiInstancesPath(Paths.get(eiHome, BACKEND_INSTANCES_DEFAULT_FILENAME).toString());
+            LOG.info("EI Instances List file path is not provided! " +
+                     "Will create a default EI Instances List file at file path: " + eiInstancesPath);
+            dumpJsonArray(backendInstancesListJsonArray);
+            ensureValidFile();
+            setDefaultEiBackendInstance(backendInstancesListJsonArray);
+            
         } else {
             setEiInstancesPath(Paths.get(backendInstancesFilePath).toString());
+            if (!(new File(eiInstancesPath).isFile())) {
+            	LOG.info("EI Instances List file don' exist! Creating file with given or " +
+                         "default EI instances list content. File path: " + eiInstancesPath);
+                createFileWithDirs();
+                dumpJsonArray(backendInstancesListJsonArray);
+                ensureValidFile();
+            }
+            
         }
     }
+    
+    private void setDefaultEiBackendInstance(JsonArray jArray) {
+    	for (JsonElement instanceJsonObj : jArray) {
+    		JsonObject jObject = instanceJsonObj.getAsJsonObject();
+    		if (Boolean.getBoolean(jObject.get("defaultBackend").toString())) {
+    			backendInstancesUtils.setDefaultBackEndInstance(jObject.get("name").toString(),
```

Comment:
Defined by the Eiffel code guidelines tabs should not be used, instead use 4 spaces. Here is a mix of tabs and spaces.

Reply:
Fixed.